services/payments/errors.go

```diff
@@ -28,3 +28,26 @@ type InsufficientAuthorizationsError struct{}
 func (e *InsufficientAuthorizationsError) Error() string {
 	return "insufficient authorizations"
 }
+
+// SolanaTransactionNotConfirmedError indicates that a Solana transaction is in any known status
```

Comment:
Just an opinion, SKUs has taken a different approach – a simple `Error` type based on `string` – https://github.com/brave-intl/bat-go/blob/master/services/skus/model/model.go#L487-L491.

The main benefit of this approach all your sentinel errors become `const`, which is very, very nice – https://github.com/brave-intl/bat-go/blob/master/services/skus/model/model.go#L24-L37.

That way, you don't have to have an inventory of types, plus it's all compilation-time.

There are some benefits of this approach with structs as well, but given the number of types you've already got, it's going to be somewhat painful over time. Overhead might not be an issue.